package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@reduxjs/toolkit": "^1.5.0",
+    "@material-ui/core": "^4.11.2",
```

Comment:
but why? If we aren't usin' it, we should toss it.

Reply:
got it!